src/DotLiquid/Hash.cs

```diff
@@ -1,15 +1,24 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using System.Linq.Expressions;
 using System.Reflection;
 
 namespace DotLiquid
 {
 	public class Hash : IDictionary<string, object>, IDictionary
-	{
-		#region Fields
+    {
```

Comment:
@haf hi, I've fixed indents in new commit. It was looking good in VS, but GH seems to have 8 spaces per tab.

It's actually Odd, that whole this project uses tabs, it's better practices to use spaces everywhere, changed my commits to Tabs thou.